doc/nrf/ug_gzll.rst

```diff
@@ -0,0 +1,466 @@
+
+.. figure:: images/gzll_fig7_host_dev_trans_ok.svg
+   :alt: Successful packet transaction
+
+   Successful packet transaction
+
+A transaction can fail if the initial packet from the Device was not received correctly by the Host, or the corresponding ACK packet was not received correctly by the Device.
+Note that packets with a failing Cyclic Redundancy Check (CRC) are ignored by Gazell.
+
+If a transaction fails the Device will attempt to retransmit the initial packet to the Host until the ACK is finally received or the maximum number of transmission attempts is reached.
+If the maximum number of transmission attempts is reached the retransmissions will stop and the nrf_gzll_device_tx_failed() callback will be called.
+
+If only the ACK packet sent from the Host to the Device is lost, but both the initial packet and the subsequent retransmission attempts are being successfully received by the Host, the repeated packets will be discarded by the Host, but the ACK packets will still be sent in return to the Device.
+This prevents the application receiving duplicate data packets at the Host.
+
+.. figure:: images/gzll_fig8_host_dev_trans_fail.svg
+   :alt: Example on failing package transaction.
+
+   Example on failing package transaction.
+
+In the figure above the maximum number of allowed transmission attempts is set to 3.
+
+Packet identification
+*********************
+
+Any packet transmitted from a Device to a Host is uniquely identified by a two bit packet ID field in the packet header together with the packet's 16-bit Cyclic Redundancy Check (CRC).
+This packet ID is used to distinguish a new packet from the previous packet if it has the same payload.
+
+On the Host side, retransmitted packets will be discarded and not added to an RX FIFO.
+
+Pipes and addressing
+====================
+
+Each logical address on the nodes is termed a pipe.
+Each pipe maps to one on-air address used when transmitting or receiving packets.
+
+The on-air addresses are composed of a 2-4 byte long "base address" in addition to a 1 byte prefix address.
+Note that the nRF5 radio uses an alternating sequence of 0s and 1s as the preamble of the packet.
+Therefore, for packets to be received correctly, the most significant byte of the base address should not be an alternating sequence of 0s and 1s, that is, it should not be 0x55 or 0xAA.
+
+Pipe 0 has its own unique base address, which is base address 0, while pipes 1-7 use the same base address, which is base address 1.
+
+Each of the 8 pipes have a unique byte-long prefix address.
+
+On-air, the most significant bit of each address byte will be transmitted first.
+The most significant byte of the 4 byte long base address is the first transmitted address byte, while the prefix byte is transmitted last.
+
+Note: The byte ordering in Gazell and the nRF5 radio peripheral are not the same, this is because the address bytes are rearranged in Gazell to match the nRF24L radios.
```

Comment:
```suggestion
.. note: 
   The byte ordering in Gazell and the nRF5 radio peripheral are not the same.
    This is because the address bytes in Gazell are rearranged to match the nRF24L radios.
```